elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -319,5 +327,29 @@ public FilterExpression visit(ComparisonNode node, Class entityType) {
 
             throw new RSQLParseException(String.format("Invalid Operator %s", op.getSymbol()));
         }
+
+        /**
+         * Returns Predicate for '=isnull=' case depending on its arguments.
+         *
+         * NOTE: Filter Expression builder specially for '=isnull=' case.
+         *
+         * @return Returns Predicate for '=isnull=' case depending on its arguments.
+         */
+        private FilterExpression buildIsNullOperator(List<Predicate.PathElement> path, List<String> arguments) {
+            Operator elideOP;
+            switch (arguments.get(0)) {
```

Comment:
Do coercion to boolean instead of checking equality against true, false, 0, and 1.  You don't handle case for example in this switch statement.